macros/base/start_print.cfg

```
@@ -22,6 +22,7 @@ gcode:
     {% set MATERIAL = params.MATERIAL|default(printer["gcode_macro _USER_VARIABLES"].print_default_material)|string %} # Material type set in the slicer
     {% set FL_SIZE = params.SIZE|default("0_0_0_0")|string %} # Get bounding box of the first layer for the adaptive bed mesh
     {% set BED_MESH_PROFILE = params.MESH|default("")|string %} # Bed mesh profile to load
+    {% set FORCE_MESH = params.FORCE_MESH|default(0)|int %}
```

I don't understant why int is better than boolean in this case?
when user pass the start_print parameter: FORCE_MESH=1 is interpreted as True, and with FORCE_MESH=0 is False??

```suggestion
    {% set force_mesh = params.FORCE_MESH|default(False) %}
```

---

Yes this should be the case, but as I said, I need to test it properly since Jinja and boolean is pretty finicky.

Here is why I prefer 0/1 from the slicer:
1. If you put FORCE_MESH=False -> this should result in False so it's ok
2. If you put FORCE_MESH="False" -> this will result is True since "False" is a non-null string so it will be very confusing and I'm not sure how the slicer will create the Gcode.

But again I know that it's finicky and that I already battled with this in the past, just not sure if it's exactly this case or if it's still the case.